<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>910</width>
    <height>1000</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>500</width>
    <height>350</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget"/>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>910</width>
     <height>25</height>
    </rect>
   </property>
   <widget class="QMenu" name="edit">
    <property name="title">
     <string>编辑(E)</string>
    </property>
    <addaction name="undoAction"/>
    <addaction name="redoAction"/>
    <addaction name="separator"/>
    <addaction name="copyAction"/>
    <addaction name="pasteAction"/>
    <addaction name="cutAction"/>
   </widget>
   <widget class="QMenu" name="file">
    <property name="title">
     <string>文件(F)</string>
    </property>
    <addaction name="newFileAction"/>
    <addaction name="openFileAction"/>
    <addaction name="openFolderAction"/>
    <addaction name="separator"/>
    <addaction name="saveFileAction"/>
    <addaction name="saveFileAsAction"/>
    <addaction name="separator"/>
    <addaction name="settingsAction"/>
   </widget>
   <widget class="QMenu" name="help">
    <property name="title">
     <string>帮助(H)</string>
    </property>
    <addaction name="aboutAction"/>
   </widget>
   <addaction name="file"/>
   <addaction name="edit"/>
   <addaction name="help"/>
  </widget>
  <widget class="QStatusBar" name="statusbar">
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="sizeGripEnabled">
    <bool>false</bool>
   </property>
  </widget>
  <action name="openFileAction">
   <property name="text">
    <string>打开文件...</string>
   </property>
   <property name="iconText">
    <string>打开文件...</string>
   </property>
   <property name="toolTip">
    <string>打开文件...</string>
   </property>
   <property name="statusTip">
    <string>从磁盘中打开文件</string>
   </property>
   <property name="font">
    <font>
     <family>Microsoft YaHei</family>
    </font>
   </property>
  </action>
  <action name="newFileAction">
   <property name="text">
    <string>新建文件</string>
   </property>
   <property name="statusTip">
    <string>新建文件</string>
   </property>
   <property name="visible">
    <bool>true</bool>
   </property>
  </action>
  <action name="saveFileAction">
   <property name="text">
    <string>保存</string>
   </property>
   <property name="statusTip">
    <string>保存文件</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="saveFileAsAction">
   <property name="text">
    <string>另存为...</string>
   </property>
   <property name="iconText">
    <string>另存为...</string>
   </property>
   <property name="toolTip">
    <string>另存为...</string>
   </property>
   <property name="statusTip">
    <string>另存为文件</string>
   </property>
  </action>
  <action name="undoAction">
   <property name="text">
    <string>撤消</string>
   </property>
   <property name="statusTip">
    <string>撤销上一步</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Z</string>
   </property>
  </action>
  <action name="redoAction">
   <property name="text">
    <string>恢复</string>
   </property>
   <property name="statusTip">
    <string>恢复下一步</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Y</string>
   </property>
  </action>
  <action name="copyAction">
   <property name="text">
    <string>复制</string>
   </property>
   <property name="statusTip">
    <string>复制选中的文本</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+C</string>
   </property>
  </action>
  <action name="pasteAction">
   <property name="text">
    <string>粘贴</string>
   </property>
   <property name="statusTip">
    <string>粘贴文本</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Y</string>
   </property>
  </action>
  <action name="cutAction">
   <property name="text">
    <string>剪切</string>
   </property>
   <property name="statusTip">
    <string>剪切选中的文本</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+X</string>
   </property>
  </action>
  <action name="aboutAction">
   <property name="text">
    <string>关于PyEditor</string>
   </property>
   <property name="shortcut">
    <string>F9</string>
   </property>
  </action>
  <action name="settingsAction">
   <property name="text">
    <string>设置</string>
   </property>
  </action>
  <action name="openFolderAction">
   <property name="text">
    <string>打开文件夹...</string>
   </property>
   <property name="iconText">
    <string>打开文件夹...</string>
   </property>
   <property name="toolTip">
    <string>打开文件夹...</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
